README.md

```diff
@@ -1,19 +1,43 @@
 # JavaScript Data Types
-In this lesson, we'll cover all of the basic data types in JavaScript.
+
+## Problem Statement
+
+JavaScript defines seven types of data, which means we need to know all of the rules of those types in order to manipulate data the way we want to.
 
 ## Objectives
-1. Define 'data type'.
-2. Perform basic type checking with the `typeof` operator.
-3. Provide a high-level overview of JavaScript's seven basic data types.
-4. Discuss interactions between data of various types in JavaScript.
 
-## What is a data type?
-At the machine level, all data on a computer are bits — 1s and 0s. Humans, it turns out, have more trouble discerning between `1100011 1100001 1110100 1110011` and `1100100 1101111 1100111 1110011` (cats and dogs) than our silicon friends. For more human-friendly representations of different pieces of information, we developed **data types**. JavaScript contains a number of operators (`+`, `!`, `<=`, etc.) and reserved words (`function`, `for`, `debugger`, etc.), which are reserved by the language for a specific purpose. ***Everything else is data***. And every piece of data falls into one of JavaScript's seven data types: numbers, strings, booleans, symbols, objects, `null`, and `undefined`.
+1. Define a data type
+2. Demonstrate basic type checking with the `typeof` operator
+3. Identify JavaScript's seven basic data types
+4. Describe interactions between data of various types in JavaScript
+
+## Define a Data Type
+
+At the machine level, all data on a computer are bits — 1s and 0s. Humans, it
+turns out, have more trouble discerning between `1100011 1100001 1110100
+1110011` and `1100100 1101111 1100111 1110011` (cats and dogs) than our silicon
+friends. For more human-friendly representations of different pieces of
+information, we developed **data types**. JavaScript contains a number of
+operators (`+`, `!`, `<=`, etc.) and reserved words (`function`, `for`,
+`debugger`, etc.), which are reserved by the language for a specific purpose.
```

Comment:
Maybe add something in here like "You'll learn more about all of these later" so people aren't thrown off by the etcs.